python-flask-audit-logs-example/app.py

```diff
@@ -0,0 +1,108 @@
+import json
+import os
+from urllib.parse import urlparse, parse_qs
+from flask import Flask, session, redirect, render_template, request, url_for
+import workos
+from datetime import datetime, timedelta
+from audit_log_events import (
+    user_signed_in,
+    user_logged_out,
+    user_connection_deleted,
+    user_organization_deleted,
+    user_organization_set,
+)
+
+# Flask Setup
+DEBUG = False
+app = Flask(__name__)
+app.secret_key = os.getenv("APP_SECRET_KEY")
+
+# WorkOS Setup
+
+workos.api_key = os.getenv("WORKOS_API_KEY")
+workos.project_id = os.getenv("WORKOS_CLIENT_ID")
+workos.base_api_url = "http://localhost:7000/" if DEBUG else workos.base_api_url
+
+# Enter Connection ID here
+
+CUSTOMER_CONNECTION_ID = "conn_01G90XQ07953ZADRSFVF1YFSXP"
```

Comment:
small nitpick - do we want an actual connection ID here?

Reply:
oh yeah thats just vestigial, removing it now